comments-by-guideline-and-success-criterion.md

```diff
@@ -1100,26 +1100,16 @@ With this substitution, it would read:
 
 **Guideline 4.1 Compatible:** Maximize compatibility with current and future <INS>**[[assistive technologies](#dfn-assistive-technologies) and accessibility features of software]**</INS>.
 
-##### parsing
+##### parsing21
 
-###### Applying SC 4.1.1 Parsing to Non-Web Documents and Software
-<div class="ednote">
-
-This section has been updated to reflect that WCAG 2.2 made 4.1.1 obsolete and removed. For WCAG 2.0 and 2.1, the 2013 WCAG2ICT guidance has been updated to reflect more current technology examples, the WCAG errata regarding content implemented using HTML and XML, and to address comments received.</div>
-
-**WCAG 2.2 Guidance:**
-<div class="note wcag2ict">
-
-WCAG 2.2 has made this success criterion obsolete and removed it as a requirement in the standard. Therefore, the interpretation of this success criterion for [non-web documents](#document) and [software](#software) has been removed.</div>
-
-**WCAG 2.0 and 2.1 Guidance:**
+###### Applying SC 4.1.1 Parsing (WCAG 2.0 and 2.1), to Non-Web Documents and Software
 
 This applies directly as written, and as described in [Intent from Understanding Success Criterion 4.1.1](https://www.w3.org/WAI/WCAG21/Understanding/parsing#intent), replacing “In content implemented using markup languages” with “For non-web documents or software that use markup languages, in such a way that the markup is separately exposed and available to assistive technologies and accessibility features of software or to a user-selectable user agent”.
 
 With this substitution, it would read:
 
 4.1.1 Parsing: **<INS>[For [non-web documents](#document) or [software](#software) that use markup languages, in such a way that the markup is separately exposed and available to [assistive technologies](#dfn-assistive-technologies) and accessibility features of software or to a user-selectable [user agent](#user-agent)]</INS>**, elements have complete start and end tags, elements are nested according to their specifications, elements do not contain duplicate attributes, and any IDs are unique, except where the specifications allow these features.
-<div class="note">
+<div class="note wcag2ict">
```

Comment:
@maryjom This seems to be a WCAG 2.0 Note.
There is disparity here. We are including the WCAG 2.0 Note and then not including the 2.1 notes. Should we be doing something about it, for example including the WCAG 2.1 Notes?

Reply:
Additional content would therefore be as follows:

<div class="note wcag2ict">
NOTE
This Success Criterion should be considered as always satisfied for any content using HTML or XML.
</div>

<div class="note wcag2ict">
NOTE
Since this criterion was written, the HTML Living Standard has adopted specific requirements governing how user agents must handle incomplete tags, incorrect element nesting, duplicate attributes, and non-unique IDs. [HTML]

Although the HTML Standard treats some of these cases as non-conforming for authors, it is considered to "allow these features" for the purposes of this Success Criterion because the specification requires that user agents support handling these cases consistently. In practice, this criterion no longer provides any benefit to people with disabilities in itself.

Issues such as missing roles due to inappropriately nested elements or incorrect states or names due to a duplicate ID are covered by different Success Criteria and should be reported under those criteria rather than as issues with 4.1.1.
</div>

Reply:
Wondering if we can address by simply saying we have replaced the WCAG notes with our notes, potentially eliminating the first one (since it originated from WCAG 2.0 and no longer exists in WCAG 2.1).

Reply:
The solution will be a little bit simpler than this thread suggests. WCAG 2.0 actually has the same erratum as WCAG 2.1. It's just that the erratum wasn't applied in the normative 2.0 publication. It is visible in Understanding SC 4.1.1 (WCAG 2.0).

I'm working on some more detailed feedback, which I will share tomorrow.

Reply:
I've already modified Daniel's PR to remove the WCAG 2.0 Note 1 that was in our guidance and saying we replaced the notes with our two notes. Wasn't planning to meet in the morning, but I can if you want to talk this out @mitchellevan as this is the only problem left to solve.

Reply:
Thanks @maryjom, let's have a quick call about the notes layout. I'll email you my availability.

Reply:
> I've already modified Daniel's PR to remove the WCAG 2.0 Note 1 that was in our guidance and saying we replaced the notes with our two notes.

After chatting with @maryjom I agree with this change.